### DISCUSSION TITLE
请问一下，为什么上传文件成功后不显示上传的文件

### DISCUSSION THREAD

**Discussion by Tomiya233** [AlistGo/alist#2516] [Q&A]
请问一下，为什么上传文件成功后不显示上传的文件，上传后阿里云盘里面显示出来了，但是AList里面仍没有
![image](https://user-images.githubusercontent.com/85816199/204236244-c2177af6-9baf-4396-bd6b-e664ef5da3f1.png)
![image](https://user-images.githubusercontent.com/85816199/204236291-16b6ea60-b354-4c72-9381-7f0ff7ac7959.png)

**Comment by Tomiya233** [comment]
求各位大佬回复

**Reply by yws112358** [reply]
页面里 点击刷新呢？

**Reply by Tomiya233** [reply]
> 页面里 点击刷新呢？
尝试过，没有用

**Reply by Tomiya233** [reply]
> 页面里 点击刷新呢？

阿里云和天翼云都是如此

**Reply by Tomiya233** [reply]
刚刚重新安装了AList，重新上传了一个文件，仍然不显示

**Comment by Tomiya233** [comment]
刚刚重新安装了AList，重新上传了一个文件，仍然不显示

**Comment by Tomiya233** [comment]
我把缓存过期时间设为0，过了一段时间，之前上传的出现了，继续上传只需刷新就显示出来了
